snuba/web/rpc/v1/resolvers/R_eap_spans/resolver_get_trace.py

```diff
@@ -86,6 +87,32 @@ def _build_query(request: GetTraceRequest) -> Query:
                 ),
             ),
         ]
+        selected_columns.extend(
+            [
+                SelectedExpression(
+                    name=col_name, expression=column(col_name, alias=col_name)
+                )
+                for col_name in [
+                    "organization_id",
+                    "project_id",
+                    "service",
+                    "trace_id",
+                    "span_id",
+                    "parent_span_id",
+                    "segment_id",
+                    "segment_name",
+                    "is_segment",
+                    "_sort_timestamp",
+                    "start_timestamp",
+                    "end_timestamp",
+                    "duration_micro",
+                    "exclusive_time_micro",
+                    "name",
+                    "sampling_factor",
+                    "sampling_weight",
```

Comment:
where is this list coming from?

Reply:
1. You are not querying the storage, you are querying the entity
2. If it's coming from the entity, you can just get the entity `get_entity(EntityKey('eap_spans')` here rather than duplicating its column names